test/test_allreduce_torchrun.py

```diff
@@ -0,0 +1,29 @@
+import os
+import subprocess
+
+def test_local_launch_allreduce():
+    # This test launches a allreduce using torchrun launcher
+    cmd = "torchrun --nproc_per_node=2  --master_addr=127.0.0.1 --master_port=2020 allreduce_torchrun.py "
+
+    proc0 = subprocess.Popen(cmd, shell=True)
+
+    return_code = proc0.wait()
+    assert return_code == 0
+
+def test_local_launch_allreduce_cores():
+    # This test launches a allreduce using torchrun launcher
+    cmd = "torchrun --nproc_per_node=2  --master_addr=127.0.0.1 --master_port=2020 allreduce_torchrun.py "
+
+    new_env0 = os.environ.copy()
+    new_env0['NEURON_RT_VISIBLE_CORES'] = '0,1'
```

Comment:
I'm not familiar with Neuron at all. Do we need to install anything extra beyond PyTorch and PyTorch/XLA to run this test? If not, can you add a GPU CI test like this? https://github.com/pytorch/xla/blob/590dee5d08ff4cf3ba72eab306b470e22bd08e16/test/run_tests.sh#L85-L90

Reply:
Yes, these are neuron specific env variables. Will add the GPU specific ones. But, we have PJRT_DEVICE listed above, will this interfere with the torchrun?

Reply:
Nope, there shouldn't be an issue. PJRT won't work with this PR, but you can add a new function to `run_tests.sh` that sets up anything neuron related and skips setting `PJRT_DEVICE` (i.e. make a `run_neuron` function like `run_pjrt`)

Reply:
Hi @will-cromar Looks like in the CI tests, the file allreduce_torchrun.py is not picked up. What is a good way to fix this? Is there a global prefix that I can add before the file name?